### PR TITLE
Added documentation for additional fields of Image

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Hi there,

As @ospfranco found there is some documentation for the code coming from #7338 missing, so I added it. It concerns the method, headers and bdoy field on the Image source object.

If you would like to have any changes made, please don't hesitate to comment, I will add them.

Have a nice day and thank you for maintaining React Native!

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @davidaurelio and @JoelMarcey to be potential reviewers.

File: docs/Images.md [hunk 1]
```diff
@@ -91,6 +91,22 @@ Many of the images you will display in your app will not be available at compile
 <Image source={{uri: 'https://facebook.github.io/react/img/logo_og.png'}} />
 ```
 
+### Network Requests for Images
+
+If you would like to set things as HTTP-Verb, Headers or a Body along with the image request you may do this by defining these properties on the source object:
```
Comment (JoelMarcey) [inline docs/Images.md]: nit: comma after "image request"

Comment (JoelMarcey) [inline docs/Images.md]: Should it be "set such things as" ?

**Comment by JoelMarcey** [comment]
Thanks. Just a couple of comments. Then we can ship.

**Comment by ospfranco** [comment]
thanks @DanielMSchmidt please bear in mind, the android counterpart of that commit has not been merged yet, [7791](https://github.com/facebook/react-native/pull/7791), so current documentation should at least mention this? or maybe it is better to hold it until it has been merged

**Comment by facebook-github-bot-2** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @davidaurelio as a potential reviewer. Could you take a look please or cc someone with more context?

**Comment by JoelMarcey** [comment]
I think we may still be waiting for #7791 to be merged here -- is that right @ospfranco?

**Comment by JoelMarcey** [comment]
The small changes I requested can be made in the meantime.

**Comment by mkonicek** [comment]
Thanks for the review @JoelMarcey 👍

**Comment by DanielMSchmidt** [comment]
I just updated your comments

**Comment by JoelMarcey** [comment]
Thanks. SO now we just need to decide whether this needs to wait for #7791

**Comment by JoelMarcey** [comment]
#7791 is still open. We are still waiting for that, right?

**Comment by ospfranco** [comment]
Hey @JoelMarcey, sorry I was short on time, yes #7791 is still open:

it was dependent on a change @rigdern made on Fresco framework for android, now that is merged so we are waiting on the pull request to be merged to react-native, once that is merged you can push this too.

**Comment by hramos** [comment]
Waiting on #7791 ...

**Comment by lacker** [comment]
Still waiting on this functionality actually getting in.

**Comment by hramos** [comment]
Waiting...

**Comment by javache** [comment]
Could you rebase this change?

**Comment by DanielMSchmidt** [comment]
@javache Did so 👍

**Pull request closed without merging**